docs/machine-learning/tutorials/object-detection-custom-vision-onnx.md

```diff
@@ -0,0 +1,388 @@
+---
+title: 'Tutorial: ML.NET prediction to detect objects in images from a Custom Vision ONNX model'
+description: Learn how to detect objects in images using an ONNX model from the Custom Vision service in ML.NET. 
+ms.date: 06/05/2022
+ms.topic: tutorial
+ms.custom: mvc, title-hack-0612
+recommendations: false
+#Customer intent: As a developer, I want to use ML.NET to categorize images using an ONNX model trained in the Custom Vision service.
+---
+# Tutorial: Categorize an image in ML.NET from Custom Vision ONNX model
+
+Learn how to use ML.NET to detect objects in images using an ONNX model trained in the Microsoft Custom Vision service.
+
+The Microsoft Custom Vision service is an AI service that allows you to upload your own images and it will train a model for you. You can then export the model to ONNX format and use it in ML.NET to make predictions.
+
+In this tutorial, you will learn how to:
+> [!div class="checklist"]
+>
+> * Understand the problem
+> * Use the Custom Vision service to create an ONNX model
+> * Incorporate the ONNX model into the ML.NET pipeline
+> * Train the ML.NET model
+> * Detect stop signs in test images
+
+A sample for the ML.NET pipeline and testing of an image can be found [here]().
+
+## Prerequisites
+
+* [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/).
+* [Download the dataset](https://aka.ms/mlnet-object-detection-tutorial-dataset) of 50 stop sign images.
+* Azure account. If you don't have one, [create a free Azure account](https://aka.ms/AMLFree).
+
+## Select the right machine learning task
+
```

Comment:
Is there something supposed to go under this section? 

A suggestion for what could go here is something similar to this where you introduce the task and explain it briefly.

https://docs.microsoft.com/dotnet/machine-learning/how-to-guides/install-ml-net-cli